reference/language-concepts/normal-ordering.markdown

```diff
@@ -70,29 +70,29 @@ make sure all needed classes and variables are determined before they are
 needed in normal evaluation, pre-evaluation takes place immediately before
 policy evaluation.
 
-In pre-evaluation files are loaded based on ordering in body common control
+During pre-evaluation files are loaded based on ordering in body common control
 (first) and body file control (after body common control). This means that
 files included in body common control are loaded and parsed before files
 placed in body file control. This is important from a common bundles
```

Comment:
Is the importance limited to *common* bundles? 

Ah yes it is. no need for change on line 76